trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -0,0 +1,35 @@
+const { param } = require('express-validator');
+
+const userParamNumericValidation = param('user').isAlphanumeric().withMessage('The User ID (user) provided should be numeric');
+const facilityIdNumericValidation = param('facilityId').isNumeric().withMessage('The Facility ID (facilityId) provided should be numeric');
+const amendmentIdNumericValidation = param('amendmentId').isNumeric().withMessage('The Amendment ID (amendmentId) provided should be numeric');
+const dealIdNumericValidation = param('dealId').isNumeric().withMessage('The Amendment ID (amendmentId) provided should be numeric');
+const partyURNNumericValidation = param('urn').isNumeric().withMessage('The party URN (urn) provided should be numeric');
```

Comment:
the format for the `urn` will be: `"00122431"`. i.e. numbers, but in quotes/string.